test_conformance/math_brute_force/FunctionList.c

```diff
@@ -175,8 +175,8 @@ const Func  functionList[] = {
                                     ENTRY_EXT( sincos,            4.0f,         4.0f,  0.00048828125f,     FTZ_OFF,    unaryF_two_results), //relaxed ulp 2^-11
                                     ENTRY( sinh,                  4.0f,         4.0f,         FTZ_OFF,     unaryF),
                                     ENTRY( sinpi,                 4.0f,         4.0f,         FTZ_OFF,     unaryF),
-                                    { "sqrt", "sqrt",     {(void*)reference_sqrt}, {(void*)reference_sqrtl}, {NULL}, 3.0f, 0.0f,    4.0f, INFINITY, FTZ_OFF, RELAXED_OFF, unaryF },
-                                    { "sqrt_cr", "sqrt",  {(void*)reference_sqrt}, {(void*)reference_sqrtl}, {NULL}, 0.0f, 0.0f,    0.0f, INFINITY, FTZ_OFF, RELAXED_OFF, unaryF },
+                                    { "sqrt", "sqrt",     {(void*)reference_sqrt}, {(void*)reference_sqrtl}, {NULL}, 3.0f, 0.0f, 0.0f,    4.0f, INFINITY, FTZ_OFF, RELAXED_OFF, unaryF },
+                                    { "sqrt_cr", "sqrt",  {(void*)reference_sqrt}, {(void*)reference_sqrtl}, {NULL}, 0.0f, 0.0f, 0.0f,   0.0f, INFINITY, FTZ_OFF, RELAXED_OFF, unaryF },
                                     ENTRY_EXT( tan,               5.0f,         5.0f,         8192.0f,    FTZ_OFF,     unaryF), //in derived mode it the ulp error is calculated as sin/cos and in non-derived mode it is the same as half_tan.
                                     ENTRY( tanh,                  5.0f,         5.0f,         FTZ_OFF,     unaryF),
                                     ENTRY( tanpi,                 6.0f,         6.0f,         FTZ_OFF,     unaryF),
```

Comment:
does this table need updated with the half precision ULPs? a quick run of `tanpi()` fp16 locally looks like it's checking against 6 ULP when it should be 2 ULP for fp16 (or 3ULP for embedded profile)